src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Security/clear_form.html.twig

```diff
@@ -0,0 +1,72 @@
+{#**
+  * Copyright since 2007 PrestaShop SA and Contributors
+  * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+  *
+  * NOTICE OF LICENSE
+  *
+  * This source file is subject to the Open Software License (OSL 3.0)
+  * that is bundled with this package in the file LICENSE.md.
+  * It is also available through the world-wide-web at this URL:
+  * https://opensource.org/licenses/OSL-3.0
+  * If you did not receive a copy of the license and are unable to
+  * obtain it through the world-wide-web, please send an email
+  * to license@prestashop.com so we can send you a copy immediately.
+  *
+  * DISCLAIMER
+  *
+  * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+  * versions in the future. If you wish to customize PrestaShop for your
+  * needs please refer to https://devdocs.prestashop.com/ for more information.
+  *
+  * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+  * @copyright Since 2007 PrestaShop SA and Contributors
+  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+  *#}
+
+<div class="row justify-content-center">
+  <div class="col form-horizontal">
+    <div class="card">
+      {% block clear_form_header %}
+        <h3 class="card-header">
+          {{ 'Outdated sessions'|trans({}, 'Admin.Advparameters.Feature') }}
+
+          <span
+            class="help-box"
+            data-container="body"
+            data-toggle="popover"
+            data-trigger="hover"
+            data-placement="right"
+            data-content="{{ 'All outdated sessions will be automatically cleared after the first connection attempt, but you can do it manually now if needed.'|trans({}, 'Admin.Advparameters.Help')|escape('html_attr') }}"
+            title=""
+            data-original-title=""
+          >
+          </span>
+        </h3>
+      {% endblock %}
+
+      {% block clear_form_content %}
+        <div class="card-block row">
+          <div class="card-text">
```

Comment:
```suggestion
        <div class="card-body">
          <div class="form-wrapper">
```

Reply:
To match with @NeOMakinG PR https://github.com/PrestaShop/PrestaShop/pull/25879 which aims to clean all cards and avoid double work or missed things